storage/mysql/admin_storage.go

```diff
@@ -301,7 +301,7 @@ func (t *adminTX) CreateTree(ctx context.Context, tree *trillian.Tree) (*trillia
 		nowMillis,
 		nowMillis,
 		privateKey,
-		newTree.PublicKey.GetDer(),
+		[]byte{}, // Unused, filling in for backward compatibility.
```

Comment:
Should this and other constants be moved directly to the SQL statement, instead of being added as parameters?

Reply:
I did something similar to this in the previous PRs, where I'm trying to touch this as little as possible, and I'd try a separate PR to tweak the SQL statements?

Reply:
Yep, separate PR sounds good.